site/ja/tutorials/keras/basic_classification.ipynb

```diff
@@ -487,17 +488,15 @@
       },
       "cell_type": "code",
       "source": [
-        "import matplotlib.pyplot as plt\n",
-        "%matplotlib inline\n",
-        "\n",
```

Comment:
matplotlib.pyplot is already imported above, so I think this part is not necessary.

Reply:
Thank you for your PR.
matplotlib.pyplot is imported in first code cell, and the magic command "%matplotlib inline" is omitted in other notebooks, so I think these lines are OK to be removed.